Makefile

```diff
@@ -2,13 +2,13 @@ login:
 	docker login
 
 build:
-	docker build -t smartbooster/docker-php-fpm .
-	docker build -t smartbooster/docker-php-fpm:builder ./builder
+	docker build -t smartbooster/php-fpm .
```

Comment:
smartbooster/php-fpm was already the name of the docker image or did you actually change it ?